OHHTTPStubs/Sources/OHHTTPStubs+Mocktail.m

```diff
@@ -0,0 +1,125 @@
+//
+//  OHHTTPStubs+Mocktail.m
+//  CardCompanion
+//
+//  Created by Wang, Sunny on 7/30/15.
+//  Copyright (c) 2015 Capital One. All rights reserved.
+//
+
+#import "OHHTTPStubs+Mocktail.h"
+
+@class OHHTTPStubsDescriptor;
+
+@implementation OHHTTPStubs (Mocktail)
+
+
++(NSArray *)stubRequestsUsingMocktailsAt:(NSURL *)dirURL{
```

Comment:
You should use a more idiomatic name for this method, like `stubRequestsUsingMocktailsAtPath:`